### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by Mtaylorr**

### CODE REVIEW

File: file3.py [hunk 1]
```diff
@@ -0,0 +1 @@
+import numpy
```
Comment (RaoufGhrissi) [inline file3.py:1]: No new line  at the end of the file

File: file4.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#printing a before declaring
+print(a)
+
+a=10
+print(a)
```
Comment (RaoufGhrissi) [inline file4.py:5]: same error

File: file4.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#printing a before declaring
+print(a)
```
Comment (RaoufGhrissi) [inline file4.py:2]: a used before declaration

File: file4.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+a=10
```
Comment (RaoufGhrissi) [inline file4.py:4]:
```suggestion
print(10) 
```
directly, no need to use additional variable

**Review by RaoufGhrissi — CHANGES_REQUESTED** [review]
@Mtaylorr  good job for your work, i added some comment

**Comment by Mtaylorr** [comment]
It's ready for review

**Pull request merged**